bin/node/testing/src/bench.rs

```diff
@@ -29,7 +29,7 @@ use crate::client::{Client, Backend};
 use crate::keyring::*;
 use sc_client_db::PruningMode;
 use sc_executor::{NativeExecutor, WasmExecutionMethod};
-use sp_consensus::{
+use sc_consensus_api::block_import::{
```

Comment:
We should keep that the same - I will re-export this from the crate so there's less changes.